README.md

```diff
@@ -10,6 +10,39 @@ To see the site live, visit https://philipcastiglione.github.io/deakin-sec-websi
 
 Join the slack channel #sec-website in DSEC slack for more.
 
+## Installing development requirements
+
+Ensure the have a recent version of [Node.js](https://nodejs.org) and `npm`.
+
+Open a terminal, and follow the instructions below.
+
+```bash
+cd /path/to/code
+git clone https://github.com/deakinsec/deakin-sec-website.git
+cd deakin-sec-website
+npm install
+```
+
```

Comment:
Probably also want to include the gulp install instructions (eg from https://gulpjs.com/) since that's required for the next step.

Reply:
Added in bcd8f8e